data/campaigns/Under_the_Burning_Suns/units/quenoth/Youth.cfg

```diff
@@ -564,6 +577,7 @@ _"Nym is Kaleh’s childhood friend, a young, rebellious lady with quick wits an
             [set_specials]
                 mode = replace
                 {WEAPON_SPECIAL_USES_ATTACKS 2}
+                {WEAPON_SPECIAL_PARRY 20}
```

Comment:
Why is `increase_parry` not getting removed here?

Reply:
opps, my bad, ti's fixed now.